Commodity_Labels_1.0/README.md

```diff
@@ -0,0 +1,10 @@
+
+**Classification 1.0**
+
+A detailed analysis can be found in the file "classification.pdf"
+
+The Python script "main.py" runs the analysis on a .json file containing scrapped articles. The script
+may give a traceback error around line 119. This happens when a 'wrong' article is selected in the analyzed sample; past 'wrong' 
```

Comment:
Add a try-catch block that takes care of this.
Also, name the file something more explicit and add a `if __name__ == "__main__":` block so that the script can be run but also imported.